src/fciqmc.jl

```diff
@@ -188,4 +188,18 @@ end
 function advance!(algorithm, report, state::ReplicaState, replica::SpectralState{1})
     return advance!(algorithm, report, state, only(replica.single_states))
 end
-# TODO: add advance! for SpectralState{N} where N > 1
+
+function advance!(algorithm, report, state::ReplicaState, replica::SpectralState{N, NS, GramSchmidt{N}}) where {N, NS}
+    proceed = true
+    if state.step[] % replica.spectral_strategy.orthogonalisation_interval == 0
+        for i in 1:N
+            for j in 1:i-1
+                add!(replica[i].v, replica[j].v, -1*dot(replica[j].v,replica[i].v)/dot(replica[j].v, replica[j].v), One())
```

Comment:
```suggestion
                add!(replica[i].v, replica[j].v, -1*dot(replica[j].v,replica[i].v)/dot(replica[j].v, replica[j].v))
```
I think that we don't need the last argument.

Also, please try to keep the length of a line below 92 characters. We are trying to adhere to the 
Invenia Blue style https://github.com/JuliaDiff/BlueStyle

Reply:
I suggest writing it as
```suggestion
                u = replica[i].v
                v = replica[j].v
                add!(u, v, -dot(u, v) / dot(v, v))
```

Reply:
Yes that looks better, thank you, I'll try to stick to that style.